### PR TITLE
Starting the catalog stuff

### PR DESCRIPTION
**Pull request by Kushagra-0801**
Just a very basic with some hard coded catalog values in it

PS: Would making the dev branch the default fix this problem of accidental pr to the wrong branch?

### CODE REVIEW

File: client/src/components/Catalog.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+class CourseTile extends Component {
+  render() {
+    return (
+      <li key={this.props.code}>
+        <ul className="course-tile">
+          <li>{this.props.code}</li>
+          <li>{this.props.name}</li>
+          <li>{this.props.midsem}</li>
+          <li>{this.props.compre}</li>
+        </ul>
```
Comment (AetherPrior) [inline client/src/components/Catalog.jsx:12]: Are you following a standard or are you suggesting we follow this?

Comment (Kushagra-0801) [reply client/src/components/Catalog.jsx:12]: Not really doing anything. This just made sense to me

**Review by VikramjeetD — APPROVED** [review]

**Comment by AetherPrior** [comment]
Shall we go ahead with the merge of this?

**Comment by Kushagra-0801** [comment]
I am merging this one in

**Pull request merged**